packages/neptune-extension/src/common/state/notifications/actions.ts

```diff
@@ -0,0 +1,21 @@
+export function addNotification(id: string, type: string, data: any) {
```

Comment:
Maybe it would be better to define Notification type here and use it as type of argument to addNotification action. It would then take an object of type Notification.

Why?

Because you already made inconsistence between these types.
addNotification has required `data` argument but `Notification` type has it optional.
So I am forced to used like this:

// I do not want to attach  any data to my notification.
`addNotification('my-id', 'my-type', undefined);`

Reply:
I've moved the type definition. Should be easier to use it now